shiptest.dme

```diff
@@ -3179,6 +3179,65 @@ g// DM Environment file for shiptest.dme.
 #include "voidcrew\code\__HELPERS\game.dm"
 #include "voidcrew\code\__HELPERS\mobs.dm"
 #include "voidcrew\code\__HELPERS\names.dm"
+#include "voidcrew\code\_globalvars\lists\flavor_misc.dm"
```

Comment:
these are all placed in the wrong place.. redo this and put them in the proper alphabetical spot. Dreammaker likes to force rearrange these and i'd rather have them all done here so someone else's PR doesn't throw this in